### ISSUE TITLE
[SUGGESTION] New fields in "Add project"

### ISSUE THREAD

**Issue by kosmo10** [graphprotocol/mission-control-curator#109]
Almost all projects use telegram and medium, I think that you need to add these fields in "Add a project" and for displaying on the page. This will give more information about the project.

**Comment by yanivtal** [comment]
Agreed these would be good fields to add. The subgraph schema is the place to add these fields. There are plans to add governance over subgraphs so that curators could vote to add fields to schemas there.

**Comment by GLCNI** [comment]
Documents and Resources (for developers, technical information), most projects have this on their main Website or Github, however there are some external or community run resources that might be worth including.
